Source/WebCore/platform/graphics/transforms/TransformState.h

```diff
@@ -42,6 +42,7 @@ class TransformState {
 public:
     enum TransformDirection { ApplyTransformDirection, UnapplyInverseTransformDirection };
     enum TransformAccumulation { FlattenTransform, AccumulateTransform };
+    enum TransformMatrixTracking { DoNotTrackTransformMatrix, TrackSVGCTMMatrix, TrackSVGScreenCTMMatrix };
```

Comment:
have you considered enum class?

Reply:
Yeah, I chose to not use it for consistency reasons. If we want to switch, then we have to move the enums out of TransformState and replace s/TransformState::ApplyTransformDirection/TransformDirection::Apply/, etc in another patch. Maybe @smfr has an opinion on that?